### ISSUE TITLE
100k Map should better reflect selected demographic group

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1390]
### PROBLEM: 
Currently the 100k map only displays the selected demographic group in the dropdown menu (and the info box if available). The dropdown box is somewhat easy to miss, and more importantly the demographic group is not reflected in the Card Title or the Vega Legend (in RED), potentially leading to a misrepresentation of the data. 

### SOLUTION:
Using the logic from the info box (in BLUE) we can add the optional selected demographic group (eg "Black or African American", "15-24") to at least the legend, and potentially also the card title. We also need to add to the A11y descriptions being created in #1389
![Screen Shot 2022-01-10 at 10 23 44 AM](https://user-images.githubusercontent.com/41567007/148811361-3c7de713-78e2-4c38-88e5-da4c27285575.png)

**Comment by benhammondmusic** [comment]
@eriwarr this would be a helpful thing to tackle if/when you're looking for a next thing to do.
